### PR TITLE
Add single build OCI 15 sp3 definition

### PR DESCRIPTION
**Pull request by smarlowucf**

### CODE REVIEW

File: images/single-cloud/oci/sles-sap-byos/content.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  packages:
+    - _attributes:
+        type: bootstrap
+      _include: base/bootstrap
+    - _attributes:
+        type: image
+      _include:
+        - base/common
+        - base/sle
+        - base/byos
```
Comment (jgleissner) [inline images/single-cloud/oci/sles-sap-byos/content.yaml:15]: There's no `base/byos`, should be removed.

**Review by jgleissner — CHANGES_REQUESTED** [review]

File: images/single-cloud/oci/sles-sap-byos/content.yaml (hunk 1 above)
Comment (smarlowucf) [reply images/single-cloud/oci/sles-sap-byos/content.yaml:15]: Ditto for the other two uses?

Comment (jgleissner) [reply images/single-cloud/oci/sles-sap-byos/content.yaml:15]: Yep.

**Comment by smarlowucf** [comment]
> We'll also need `drivers` sections under `image` (below `packages` I'd say) that include `csp/oci`.

This one is confusing. If I grep for any usage of "drivers:" in the images dir there are no matches. I would assume there'd be examples to go off of? `grep -rnw images/ -e 'drivers:'`

**Comment by jgleissner** [comment]
> > We'll also need `drivers` sections under `image` (below `packages` I'd say) that include `csp/oci`.
> 
> This one is confusing. If I grep for any usage of "drivers:" in the images dir there are no matches. I would assume there'd be examples to go off of? `grep -rnw images/ -e 'drivers:'`

OCI is the only image that uses `drivers`. We'll need:
```
image:
  drivers:
    - _include:
      - csp/oci
```

File: images/single-cloud/oci/sles-byos/15-sp3/image.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+include-paths:
+  - sle15/sp3
```
Comment (jgleissner) [inline images/single-cloud/oci/sles-byos/15-sp3/image.yaml:2]: `include-paths` should also include `sle15/sp2` and `sle15/sp1` to inherit from older SPs where applicable. Sorry, missed that earlier.

File: images/single-cloud/oci/sles-sap-byos/15-sp3/image.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+include-paths:
+  - sle15/sp3
```
Comment (jgleissner) [inline images/single-cloud/oci/sles-sap-byos/15-sp3/image.yaml:2]: Same as above.

**Review by jgleissner — CHANGES_REQUESTED** [review]

**Review by jgleissner — APPROVED** [review]
Looks good, thanks!

**Review by rjschwei — APPROVED** [review]

**Pull request merged**